setup.py

```diff
@@ -53,11 +53,7 @@ def get_version():
     description="Python based FIDO 2.0 library",
     url="https://github.com/Yubico/python-fido2",
     python_requires=">=2.7.6,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*",
-    install_requires=[
-        "six",
-        "cryptography>=1.5",
-        'uhid-freebsd>=1.2.1;platform_system=="FreeBSD"',
-    ],
+    install_requires=["six", "cryptography>=1.5",],
```

Comment:
I think getting rid of the trailing comma inside the list should make both flake8 and black happy.

Reply:
Yes, that lets tests pass.